pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -1003,13 +1027,45 @@ func (r *Reconciler) runReconcileSeedFlow(
 				Name: "Deploying ETCD Druid",
 				Fn:   etcdDruid.Deploy,
 			})
+			_ = g.Add(flow.Task{
+				Name: "Deploying Fluent Operator Resources and wait until become ready",
+				Fn:   component.OpWait(fluentOperatorResources).Deploy,
+			})
+			_ = g.Add(flow.Task{
+				Name: "Deploying Fluent Operator",
+				Fn:   component.OpWait(fluentOperator).Deploy,
+			})
 		)
 	}
 
 	if err := g.Compile().Run(ctx, flow.Opts{Log: log}); err != nil {
 		return flow.Errors(err)
 	}
 
+	if loggingEnabled {
+		// Waiting for fluent-bit service to be created
+		fluentBitService := &corev1.Service{}
+		if err := retry.UntilTimeout(ctx, 5*time.Second, time.Minute, func(ctx context.Context) (done bool, err error) {
+			if err = seedClient.Get(ctx, kubernetesutils.Key(v1beta1constants.GardenNamespace, v1beta1constants.DaemonsetNameFluentBit), fluentBitService); err != nil {
+				return retry.MinorError(fmt.Errorf("%s service is not ready: %v", v1beta1constants.DaemonsetNameFluentBit, err))
+			}
+			return retry.Ok()
+		}); err != nil {
+			return err
+		}
+
+		// Patching fluent-bit service with the network policy annotations
+		fluentBitServicePatch := client.MergeFrom(fluentBitService.DeepCopy())
+		fluentBitService.ObjectMeta.Annotations = map[string]string{
+			v1beta1constants.AnnotationNetworkPolicyFromPolicyPodLabelSelector: v1beta1constants.NetworkPolicyAllScrapeTargets,
```

Comment:
The fluent-bit Service currently has the annotation
```yaml
  networking.resources.gardener.cloud/from-policy-pod-label-selector: all-seed-scrape-targets
```

https://github.com/gardener/gardener/blob/ba890b82d5935d1e39152efde36153e602a8724d/charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-service.yaml#L8

while this patch is adding the annotation

```yaml
    networking.resources.gardener.cloud/from-policy-pod-label-selector: all-scrape-targets
```

Is that expected?

Reply:
No, this is my mistake.